BackofficeBundle/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -246,3 +246,31 @@
           data-url="{{ path('open_orchestra_backoffice_internal_link_form') }}"
       ></div>
 {% endblock %}
+{% block oo_role_choice_widget %}
```

Comment:
the form is more readable but still a lot of space, can use an inline component ?